scripts/load-breaches.js

```diff
@@ -41,14 +41,17 @@ async function handleBreachesResponse(response) {
 }
 
 (async () => {
-  if (args["--createAMBreach"]) {
-    await DBUtils.createBreach("AllMusic", {
-      Name: "AllMusic",
-      BreachDate: "2015-012-06",
-      DataClasses: ["Email addresses", "IP addresses", "Passwords", "Usernames", "Website activity"],
-      PwnCount: 1436486,
-    });
-  }
+  // if (args["--createAMBreach"]) {
```

Comment:
nit: change this to `--createExampleBreach` to match L22 above.

Reply:
Well well well. This explains why I was having such a bad time trying to get the Example Breach to load.